howtos/environment.md

```diff
@@ -5,9 +5,10 @@ As this is a public repo, everything is stored in local environment keys.
 #### Environment Settings:
  - Contenful, see `config/contentful.rb`
    - Create your **own** Contentful keys for each space.
-   - Blog Space: https://app.contentful.com/spaces/91sm3pewxzag/api/keys
+   <!-- THE Blog is not deprecated - and built as part of the gatsby site static-www -->
+   <!-- - Blog Space: https://app.contentful.com/spaces/91sm3pewxzag/api/keys
      - CONTENTFUL_BLOG_ACCESS_TOKEN
-     - CONTENTFUL_BLOG_PREVIEW_TOKEN
+     - CONTENTFUL_BLOG_PREVIEW_TOKEN -->
```

Comment:
Oh…does this work?  I've never _commented out_ stuff in markdown lol.

I might even just keep it, but keep the deprecation note (eg. don't we point to this file on `static-www`?)

Reply:
oh righto - yup will do that!